docs/index.asciidoc

```diff
@@ -363,6 +336,21 @@ Instruct the socket to use TCP keep alive. If it's `true` then the underlying so
 will use the OS defaults settings for keep alive. If it's `false` it doesn't configure any
 keep alive setting for the underlying socket.
 
+[id="plugins-{type}s-{plugin}-obsolete-options"]
+==== TCP Input Obsolete Configuration Options
+
+WARNING: As of version `7.0.0` of this plugin, the following configuration options are no longer available,
```

Comment:
WARNING: As of version `7.0.0` of this plugin, some configuration options have been replaced.
The plugin will fail to start if it contains any of these obsolete options.